x/evm/types/access_list_tx.go

```diff
@@ -0,0 +1,267 @@
+// Copyright 2021 Evmos Foundation
+// This file is part of Evmos' Evmos packages.
+//
+// The Evmos packages is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Evmos packages is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Evmos packages. If not, see https://github.com/evmos/ethermint/blob/main/LICENSE
+package types
+
+import (
+	"math/big"
+
+	errorsmod "cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
+	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
+
+	// TODO: remove once Ethermint is migrated to Evmos
+	evmtypes "github.com/evmos/ethermint/x/evm/types"
+
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/evmos/ethermint/types"
```

Comment:
Q: Should this be imported too?

Reply:
Haven't migrated the types package yet.. only the /evm/types